src/components/new-safe/CreateSafe/useSyncSafeCreationStep.ts

```diff
@@ -4,20 +4,22 @@ import type { StepRenderProps } from '@/components/new-safe/CardStepper/useCardS
 import type { PendingSafeData } from '@/components/new-safe/steps/Step4'
 import type { NewSafeFormData } from '@/components/new-safe/CreateSafe/index'
 import { SAFE_PENDING_CREATION_STORAGE_KEY } from '@/components/new-safe/steps/Step4'
+import useWallet from '@/hooks/wallets/useWallet'
 
-const useSetCreationStep = (setStep: StepRenderProps<NewSafeFormData>['setStep'], isConnected: boolean) => {
+const useSyncSafeCreationStep = (setStep: StepRenderProps<NewSafeFormData>['setStep']) => {
```

Comment:
I think even though it's quite trivial we should write a small test for this.

Reply:
done